src/components/AssetTypesComponent.jsx

```diff
@@ -16,41 +16,19 @@ import { loadAssetTypes } from '../_actions/assetTypes.actions';
 export class AssetTypesComponent extends React.Component {
   state = {
     activePage: 1,
-    limit: 10,
-    offset: 0,
-    currentAssetTypes: []
+    limit: 10
   }
   componentDidMount() {
-    this.props.loadAssetTypes();
-    this.setCurrentAssetTypes();
+    this.props.loadAssetTypes(this.state.activePage);
   }
 
-  componentDidUpdate(prevProps) {
-    if (this.props.assetTypes.length !== prevProps.assetTypes.length) {
-      this.setCurrentAssetTypes();
-    }
-  }
-
-  getOffset = (activePage, limit) => (activePage - 1) * limit;
-
-  setCurrentAssetTypes = () => {
-    const currentAssetTypes = this.props.assetTypes.slice(
-      this.state.offset,
-      (this.state.activePage * this.state.limit)
-    );
-    this.setState({
-      currentAssetTypes
-    });
-  };
-
   handlePaginationChange = (e, { activePage }) => {
     this.setState({
-      activePage,
-      offset: this.getOffset(activePage, this.state.limit)
-    }, () => this.setCurrentAssetTypes());
+      activePage
+    }, () => this.props.loadAssetTypes(this.state.activePage));
```

Comment:
Same as the previous comment, the setState callback is unnecessary. Please change to:
```
handlePaginationChange = (e, { activePage }) => {
  this.setState({ activePage });
  this.props.loadAssetTypes(activePage);
}
```